setuptools/extern/__init__.py

```diff
@@ -71,6 +71,6 @@ def install(self):
 
 names = (
     'packaging', 'pyparsing', 'ordered_set', 'more_itertools', 'importlib_metadata',
-    'zipp', 'importlib_resources', 'jaraco', 'typing_extensions', 'nspektr', 'tomli',
+    'zipp', 'importlib_resources', 'jaraco', 'typing_extensions', 'tomli',
```

Comment:
It's possible that removing nspektr also removes dependencies of nspektr. Let's confirm and remove them also if appropriate.

Reply:
Will double check.